### PR TITLE
[IMP]12573: make custom color for fsm stage will reflect to geoengine mapping

### PR DESCRIPTION
**Pull request by smangukiya**
Modified module:
> fieldservice

Included features:
1. Custom color support for geoengine mapping
2. Increase marker/pointer size in base_geoengine mapping
3. Custom color support for gantt/web_timeline view with fsm.stage color

Note: This change will only work once we include PR https://github.com/asaunier/geospatial/pull/1 with base_geoengine repo.

### CODE REVIEW

File: fieldservice/README.rst [hunk 1]
```diff
@@ -47,6 +47,15 @@ To configure this module, you need to:
...
 
+You need to add attribute mention below with the tag <timeline> as base element.
+
+* colors (optional): it allows to set certain specific colors if the expressed
+  condition (JS syntax) is met.
+* custom_color (optional): it allows to set custom color for fsm.stages
+  example custom_color = "true". And there is minor condition to follow to
+  implement this as. Define any one stage color condition like 
+  colors="#ffffff:stage_id=='New';"
+
```
Comment (max3903) [inline fieldservice/README.rst]: @smangukiya The README.rst is generated from the files in the `readme` directory. This content needs to be moved into `readme/CONFIGURE.rst`.

File: fieldservice/models/fsm_stage.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2018 - TODAY, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import fields, models, api
```
Comment (max3903) [inline fieldservice/models/fsm_stage.py]: @smangukiya Can you put the import in alphabetic order?

**Review by max3903 — CHANGES_REQUESTED** [review]
@smangukiya Can you also fix the PEP8 errors:
https://travis-ci.org/OCA/field-service/jobs/469893579#L783

File: fieldservice/README.rst (hunk 1 above)
Comment (smangukiya) [reply fieldservice/README.rst]: Done

File: fieldservice/models/fsm_stage.py (hunk 2 above)
Comment (smangukiya) [reply fieldservice/models/fsm_stage.py]: Done

**Review by max3903 — APPROVED** [review]

**Pull request merged**